doc/zh-CN/source/2.introduction/5.architecture.md

```diff
@@ -4,12 +4,12 @@
 
 ## 1.简介
 
-![产品架构](../../images/architecture.png)
+![产品架构](../../../images/architecture.png)
```

Comment:
there is a symbolic link doc/zh-CN/images to doc/images, so no need to modify this path

Reply:
currently the link is working as expected: https://tugraph-db.readthedocs.io/zh_CN/latest/2.introduction/5.architecture.html

Reply:
OK, it works.

I create this pr by link from readthedocs edit link for test.